docs/content/audit-policy.md

```diff
@@ -4,10 +4,10 @@ title: "Audit policy"
 
 Kubernetes auditing provides a security-relevant, chronological set of records documenting the sequence of actions in a
 cluster. The cluster audits the activities generated by users, by applications that use the Kubernetes API, and by the
-control plane itself. The `auditpolicypatch` external patch will generate appropriate configuration for the Kubernetes
+control plane itself. The `clusterconfigpatch` external patch will generate appropriate configuration for the Kubernetes
 control plane.
 
-To enable the audit policy enable the `auditpolicypatch` external patch on `ClusterClass`.
+To enable the meta handler enable the `clusterconfigvars` and `clusterconfigpatch` external patches on `ClusterClass`.
```

Comment:
Actually this would be provider specific eg awsclusterconfig which includes the necessary variables. I think we should drop the individual patch docs and only have the provider specific docs as this project only exposes them with this pr

Reply:
Going to delete this commit, we can update the docs in a separate PR.

Reply:
Thanks for doing it anyway!